Dropping CUDA 11 Support in v25.08 (#607)
rapidsai/docs: _notices/rsn0048.md

---
layout: notice
parent: RAPIDS Support Notices
grand_parent: RAPIDS Notices
nav_exclude: true
notice_type: rsn
# Update meta-data for notice
notice_id: 48 # should match notice number
notice_pin: true # set to true to pin to notice page
title: "Deprecation announcement for CUDA 11 in v25.08"
notice_author: RAPIDS Ops
notice_status: "In Progress
notice_status_color: yellow
# 'notice_status' and 'notice_status_color' combinations:
#   "Proposal" - "blue"
#   "Completed" - "green"
#   "Review" - "purple"
#   "In Progress" - "yellow"
#   "Closed" - "red"
notice_topic: Platform Support Change
notice_rapids_version: "v25.08"
notice_created: 2025-05-21
# 'notice_updated' should match 'notice_created' until an update is made

> notice_updated: 2025-05-21

Reviewer: Are we still happy with these dates?

---

## Overview

> RAPIDS is dropping all CUDA 11 support, including containers and all published packages (wheels and conda) in Release `v25.08`, scheduled for August 7, 2025. `v25.06` will be the last RAPIDS release to support CUDA 11 runtimes in any format. We are continuing support for CUDA 12.0 and 12.9 containers.

Reviewer 2: Agreed. Committing this.

> RAPIDS is dropping all CUDA 11 support, including containers and all our published packages (wheels and conda)in Release `v25.08`, scheduled for August 7, 2025. `25.06` will be the last RAPIDS release to support CUDA 11 runtimes in any format. We are continuing support for CUDA 12.8 and 12.9 containers.

Reviewer 2: ```suggestion
RAPIDS is dropping all CUDA 11 support, including containers and all published packages (wheels and conda) in Release `v25.08`, scheduled for August 7, 2025. `v25.06` will be the last RAPIDS release to support CUDA 11 runtimes in any format. We are continuing support for CUDA 12.0 and 12.9 containers.
```

## Impact

> Effective RAPIDS `v25.08` release, RAPIDS will cease distribution of CUDA 11, including containers and all our published packages (wheels and conda).

Reviewer 3: ```suggestion
Effective RAPIDS `v25.08` release, RAPIDS will cease distribution of CUDA 11 release artifacts, including containers and all our published packages (wheels and conda).
```

> RAPIDS will support CUDA 12.8 and 12.9 Docker containers.

Reviewer 2: ```suggestion
RAPIDS will support CUDA 12.0 and 12.9 Docker containers.
```
